From 976a05813aa9495830a14b7acdb70a6da13da783 Mon Sep 17 00:00:00 2001
From: AndyChi <andy.chi@canonical.com>
Date: Tue, 13 Dec 2022 23:19:27 +0800
Subject: [PATCH] hwdb: Add additional Dell models that require
 ACCEL_LOCATION=base (#25724)

(cherry picked from commit e4970f5dae9657ef18abeaf0990afb47bda407e2)

Related: RHEL-5950
---
 hwdb.d/60-sensor.hwdb | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/hwdb.d/60-sensor.hwdb b/hwdb.d/60-sensor.hwdb
index 56e321880a..89119247fa 100644
--- a/hwdb.d/60-sensor.hwdb
+++ b/hwdb.d/60-sensor.hwdb
@@ -339,6 +339,8 @@ sensor:modalias:platform:HID-SENSOR-200073:dmi:*svnDell*:sku0B09:*
 sensor:modalias:platform:HID-SENSOR-200073:dmi:*svnDell*:sku0B0B:*
 sensor:modalias:platform:HID-SENSOR-200073:dmi:*svnDell*:sku0B0D:*
 sensor:modalias:platform:HID-SENSOR-200073:dmi:*svnDell*:sku0B11:*
+sensor:modalias:platform:HID-SENSOR-200073:dmi:*svnDell*:sku0C00:*
+sensor:modalias:platform:HID-SENSOR-200073:dmi:*svnDell*:sku0C02:*
  ACCEL_LOCATION=base
 
 # Dell Venue 8 Pro 3845
